advisories/github-reviewed/2025/08/GHSA-w2cq-g8g3-gm83/GHSA-w2cq-g8g3-gm83.json

```diff
@@ -52,14 +52,19 @@
       "type": "WEB",
       "url": "https://github.com/helmetjs/content-security-policy-parser/commit/b13a52554f0168af393e3e38ed4a94e9e6aea9dc"
     },
+    {
+      "type": "ADVISORY",
+      "url": "https://github.com/advisories/GHSA-m4gq-x24j-jpmf"
+    },
     {
       "type": "PACKAGE",
       "url": "https://github.com/helmetjs/content-security-policy-parser"
     }
   ],
   "database_specific": {
     "cwe_ids": [
-      "CWE-1321"
+      "CWE-1321",
+      "CWE-1395"
```

Comment:
I don't think CWE-1395 applies here. I think that refers to vulnerable dependencies, which was unrelated to this issue.